src/pages/upgrades/get-involved/index.tsx

```diff
@@ -455,7 +445,12 @@ const GetInvolvedPage = ({
         {getClientCards("cl")}
       </Content>
       <Staking>
-        <StyledCalloutBanner
+        <CalloutBanner
+          bg="transparent"
+          w={{ base: "full", md: "null" }}
+          p={{ base: 0, md: "null" }}
+          pt={{ base: 16, md: "null" }}
+          ml={{ base: 0, md: "null" }}
```

Comment:
`null` is not a valid css value.

I think it should be something like:
```suggestion
          w={{ base: "full" }}
          p={{ base: 0, sm: 12 }}
          pt={{ base: 16, sm: 12 }}
          ml={{ base: 0 }}
```